hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/UINotification.java

```diff
@@ -61,11 +63,10 @@ public void displayWarning(final String message) {
      *            as message.
      */
     public void displayValidationError(final String message) {
-        final StringBuilder updatedMsg = new StringBuilder(FontAwesome.EXCLAMATION_TRIANGLE.getHtml());
-        updatedMsg.append(' ');
-        updatedMsg.append(message);
-        notificationMessage.showNotification(SPUIStyleDefinitions.SP_NOTIFICATION_ERROR_MESSAGE_STYLE, null,
-                updatedMsg.toString(), true);
+        final StringBuilder updatedMsg = new StringBuilder(message);
+        notificationMessage.setIcon(FontAwesome.EXCLAMATION_TRIANGLE);
+        notificationMessage.showNotification(SPUIStyleDefinitions.SP_NOTIFICATION_ERROR_MESSAGE_STYLE, updatedMsg.toString(), null
```

Comment:
please set updateMsg as a notification description rather than caption

Reply:
As discussed it will stay in the caption.